src/features/projectsV2/ProjectDetails/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { useContext, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import ProjectSnippet from '../ProjectSnippet';
+import { useProjects } from '../ProjectsContext';
+import ProjectInfoSection from './components/ProjectInfoSection';
+import { getRequest } from '../../../utils/apiRequests/api';
+import { useTenant } from '../../common/Layout/TenantContext';
+import { useLocale } from 'next-intl';
+import { handleError, APIError, ClientError } from '@planet-sdk/common';
+import { ErrorHandlingContext } from '../../common/Layout/ErrorHandlingContext';
+import styles from './ProjectDetails.module.scss';
+import Skeleton from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
+import { ExtendedProject } from '../../common/types/projectv2';
+
+const ProjectDetails = ({
+  currencyCode,
+  isMobile,
+}: {
+  currencyCode: string;
+  isMobile: boolean;
+}) => {
+  const {
+    singleProject,
+    setSingleProject,
+    setIsLoading,
+    setIsError,
+    setSelectedMode,
+  } = useProjects();
+  const { setErrors, redirect } = useContext(ErrorHandlingContext);
+  const { tenantConfig } = useTenant();
+  const locale = useLocale();
+  const router = useRouter();
+  const projectSlug = router.query.p;
+
+  useEffect(() => {
+    async function loadProject(
+      projectSlug: string,
+      locale: string,
+      currency: string
+    ) {
+      setIsLoading(true);
+      setIsError(false);
+      try {
+        const fetchedProject = await getRequest<ExtendedProject>(
+          tenantConfig.id,
+          `/app/projects/${projectSlug}`,
+          {
+            _scope: 'extended',
+            currency: currency,
+            locale: locale,
+          }
+        );
+        const { purpose } = fetchedProject;
+        if (purpose === 'conservation' || purpose === 'trees') {
+          setSingleProject(fetchedProject);
+        } else {
+          throw new ClientError(404, {
+            error_type: 'project_not_available',
+            error_code: 'project_not_available',
+          });
+        }
+      } catch (err) {
+        setErrors(handleError(err as APIError | ClientError));
+        setIsError(true);
+        redirect('/');
+      } finally {
+        setIsLoading(false);
+      }
+    }
+
+    if (typeof projectSlug === 'string' && currencyCode)
+      loadProject(projectSlug, locale, currencyCode);
+  }, [projectSlug, locale, currencyCode]);
+
+  return singleProject ? (
+    <div className={styles.projectDetailsContainer}>
+      <ProjectSnippet
+        project={singleProject}
+        showTooltipPopups={false}
+        isMobile={isMobile}
+        page="project-details"
+      />
+      <ProjectInfoSection
+        project={singleProject}
+        isMobile={isMobile}
+        setSelectedMode={setSelectedMode}
+      />
+    </div>
+  ) : (
+    <Skeleton className={styles.projectInfoSkeleton} />
```

Comment:
Since we have a ProjectInfoSection component, the `projectInfoSkeleton` class could be renamed to `projectDetailsSkeleton` as that is what it appears as a placeholder for